jsk_panda_robot/panda_eus/models/dual_panda2.urdf.xacro

```diff
@@ -0,0 +1,285 @@
+<?xml version="1.0"?>
```

Comment:
humm, is  dual_panda and dual_panda2 different? do they have different gripper?

Reply:
They have different sensors.
dual_panda1: xtion camera on head, web cameras on both hands, microphone on right hand
dual_panda2: realsense d435 on both hands